x-pack/auditbeat/fips.go

```diff
@@ -0,0 +1,10 @@
+// Copyright Elasticsearch B.V. and/or licensed to Elasticsearch B.V. under one
+// or more contributor license agreements. Licensed under the Elastic License;
+// you may not use this file except in compliance with the Elastic License.
+
+//go:build requirefips
+
+// enforce fips140=only when running in fips mode
```

Comment:
Consider making the comment explain what `fips140=only` does. Assuming IIUC... maybe

`// In FIPS mode, make cryptographic algorithms that are not FIPS 140-3 compliant return an error or panic.`

Reply:
Suggest linking to https://go.dev/doc/godebug#default and https://go.dev/doc/godebug#go-124.